src/ua_timer.h

```diff
@@ -33,6 +33,8 @@ typedef struct UA_TimerEntry {
                                                 the interval is zero, the
                                                 callback is not repeated and
                                                 removed after execution. */
+    UA_Boolean handleCycleMissWithBaseTime;  /* Handle the cycle misses with the next
```

Comment:
The `pahole` tool gives the current alignment of members as this:
```c
struct UA_TimerEntry {
        struct {
                struct UA_TimerEntry * zip_left;         /*     0     8 */
                struct UA_TimerEntry * zip_right;        /*     8     8 */
                unsigned char      rank;                 /*    16     1 */
        } zipfields;                                     /*     0    24 */

        /* XXX last struct has 7 bytes of padding */

        UA_DateTime                nextTime;             /*    24     8 */
        UA_UInt64                  interval;             /*    32     8 */
        UA_ApplicationCallback     callback;             /*    40     8 */
        void *                     application;          /*    48     8 */
        void *                     data;                 /*    56     8 */
        /* --- cacheline 1 boundary (64 bytes) --- */
        struct {
                struct UA_TimerEntry * zip_left;         /*    64     8 */
                struct UA_TimerEntry * zip_right;        /*    72     8 */
                unsigned char      rank;                 /*    80     1 */
        } idZipfields;                                   /*    64    24 */

        /* XXX last struct has 7 bytes of padding */

        UA_UInt64                  id;                   /*    88     8 */

        /* size: 96, cachelines: 2, members: 8 */
        /* paddings: 2, sum paddings: 14 */
        /* last cacheline: 32 bytes */
};
```
If we put `handleCycleMissWithBaseTime` in the current location, we would move regularly used members behind the cacheline. This comes with a significant performance penalty in a "hot path" of the stack.
Please move `handleCycleMissWithBaseTime` immediately behind the `zipfields` members to use up the padding space instead.

The wording `handleCycleMissWithBaseTime` does not say *how* the missed cycles are handled.
Since we have 7 bytes of padding available, we can make the whole thing more explicit.
Define an enum `UA_TimerPolicy` instead of using a boolean.
And call the variables `timerPolicy` in the function arguments. Its more explicit and shorter.